src/layout/BaseLayout/_base-layout.scss

```diff
@@ -51,6 +51,7 @@
   // Override the Vanilla default that sets overflow-y to auto but this prevents
   // sticky from being used by the the entity details sidebar.
   overflow-y: visible !important;
+  overflow-x: scroll;
```

Comment:
Making the scrolling change here means that everything in the content scrolls, including the header:

<img width="858" alt="Screenshot 2023-08-01 at 10 08 12 am" src="https://github.com/canonical/juju-dashboard/assets/361637/334d3230-dedb-4985-9eda-c3883d1a27bd">

The other tables get wrapped in an element with `u-overflow--auto` so that only the table scrolls (I'd be keen for us to investigate if we can move away from horizontal scrolling of the tables at some stage but it would require global changes). I've made another comment about where to add this.